horizons/world/playerstats.py

```diff
@@ -117,12 +117,14 @@ def _collect_info(self):
 			TIER.PIONEERS: 3,
 			TIER.SETTLERS: 7,
 			TIER.CITIZENS: 15,
+			TIER.MERCHANTS: 20, # FIXME: dont know what valve must be here
```

Comment:
These were added in c6be172105a112399bddb3b10d3c6e63277932f1, the commit message suggests that they need proper balancing.

I'd say the values look fine, you can remove the FIXME.